spec/parallel_tests/test/runner_spec.rb

```diff
@@ -108,7 +108,18 @@ def call(*args)
         ParallelTests::Test::Runner.should_receive(:runtimes).
           and_return({"aaa" => 0, "bbb" => 3, "ccc" => 1, "ddd" => 2})
         result = call(["aaa", "bbb", "ccc", "ddd", "eee"], 3, isolate: true, single_process: [/^aaa/], group_by: :runtime)
-        result.should == [["aaa"], ["bbb", "eee"], ["ccc", "ddd"]]
```

Comment:
I have no idea how you are getting the originally expected. I'm using MRI 2.2.1.

I think the tests should be immune to ordering regardless - just like `match_array` matcher should be used to test array contents. If anyone ever tweaks the algorithm, tests shouldn't fail.

Also, testing grouping with same-size items should be tested anyway.